lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -549,9 +605,9 @@ class WebMapServiceCatalogItem
 
   @computed
   private get _currentImageryParts(): ImageryParts | undefined {
-    const imageryProvider = this._createImageryProvider(
-      this.currentDiscreteTimeTag
-    );
+    const imageryProvider = this._createImageryProvider({
+      time: this.currentDiscreteTimeTag
```

Comment:
I think this change might cause layers to flicker and be re-created when the opacity slider is moved.

Reply:
I have reverted `_createImageryProvider` to the old style accepting only time param. Instead i'm picking extra params inside `_createImageryProvider` if it is showing diff.